packages/manager/modules/exchange/src/domain/dkim-autoconfig/domain-dkim-autoconfig.html

```diff
@@ -4,6 +4,7 @@
         data-wizard
         data-wizard-on-cancel="resetAction"
         data-wizard-on-finish="ctrl.onFinishDkim.bind(ctrl)"
+        data-wizard-cancel-button-text="ctrl.hideConfirmButton ? 'exchange_tab_domain_diagnostic_dkim_cancel' : 'exchange_tab_domain_diagnostic_dkim_close' | translate"
```

Comment:
Same as https://github.com/ovh/manager/pull/10422#discussion_r1410245519

Reply:
Same. 

I will test it again and make changes if fails.